Example/Podfile

```diff
@@ -29,6 +29,15 @@ post_install do |installer|
     config.build_settings.delete('CODE_SIGNING_REQUIRED')
   end
 
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+        if target.name == "Backpack" || target.name == "FloatingPanel"
+            config.build_settings['SWIFT_VERSION'] = '5.0'
```

Comment:
Without this the pods reset to the previous swift versions every time we do `pod install` and it turns into a compile failure. [Some info about this](https://jeroenscode.com/migrating-to-swift-5/).

Reply:
There's a way to indicate swift version via the `.podspec`. I feel like we should be using that and check if FloatingPanel hasn't added a way to do this too?